src/assets/more.svg

```diff
@@ -0,0 +1,6 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="32" height="32" viewBox="0 0 32 32">
```

Comment:
Where `more.svg` file uses? Icon path defined in `src/services/upper-bar/ui/more/more.component.tsx` and rendered with `Icon` component